src/remote/remote.go

```diff
@@ -240,8 +240,9 @@ func (c *Client) Build(tid int, target *core.BuildTarget) (*core.BuildMetadata,
 	if err != nil {
 		return metadata, err
 	}
-	hash, _ := hex.DecodeString(c.digestMessage(ar).Hash)
 	if c.state.TargetHasher != nil {
+		log.Warning("here %s %s", target, ar)
```

Comment:
o.O

Reply:
never happened

Reply:
Ahaha and you actually did a -f so it really never happened